lib/kitchen/provisioner/policyfile_zero.rb

```diff
@@ -96,9 +96,10 @@ def create_sandbox
       # (see Base#run_command)
       def run_command
         level = config[:log_level] == :info ? :auto : config[:log_level]
-        chef_client_bin = sudo(config[:chef_client_path])
+        
+        cmd = "#{sudo(config[:chef_client_path])} --local-mode".
+          tap { |str| str.insert(0, "& ") if powershell_shell? }
```

Comment:
Oh boy...  I hope the arguments don't contain any $ or '.  Not that this fix breaks anything that wouldn't already be broken but I get scared anytime we just quote a string and hand it over to powershell due to the general hilarity of PS argument parsing.

Oh well...